src/Environment/Env.php

```diff
@@ -69,10 +72,9 @@ public function __construct()
             break;
         }
 
-        /** @var \Ps_checkout $module */
-        $module = \Module::getInstanceByName('ps_checkout');
-
-        $this->setMode($module->getService('ps_checkout.paypal.configuration')->getPaymentMode());
+        $this->setMode(
+            Mode::LIVE === Configuration::get(PayPalConfiguration::PAYMENT_MODE) ? Mode::LIVE : Mode::SANDBOX
```

Comment:
Please use ShopId parameter to retrieve Configuration value
I think we can use ConfigurationAdapter here